runtime/compiler/optimizer/J9RecognizedCallTransformer.cpp

```diff
@@ -139,10 +139,19 @@ bool J9::RecognizedCallTransformer::isInlineable(TR::TreeTop* treetop)
       {
       case TR::sun_misc_Unsafe_getAndAddInt:
       case TR::sun_misc_Unsafe_getAndSetInt:
-         return TR::Compiler->target.cpu.isX86() && !comp()->getOption(TR_DisableUnsafe) && !comp()->compileRelocatableCode() && !TR::Compiler->om.canGenerateArraylets();
+         return (TR::Compiler->target.cpu.isX86()
+                       || TR::Compiler->target.cpu.isPower())
+                    && !comp()->getOption(TR_DisableUnsafe)
+                    && !comp()->compileRelocatableCode()
+                    && !TR::Compiler->om.canGenerateArraylets();
```

Comment:
These queries will need updating after https://github.com/eclipse/omr/pull/2999 is merged. I suggest we mark this PR as WIP until this is done.